src/packager.ts

```diff
@@ -99,6 +101,7 @@ export async function download(
     const localFilename = join(stagingDir, input.filename);
     const { status, stderr } = spawnSync('aws', [
       's3',
+      endpointUrl ? `--endpoint-url=${endpointUrl}` : '',
```

Comment:
It seems this will fail if no endpoint is provided, since the empty string will be interpreted as an unknown argument.

```
> node -e "const {spawnSync} = require('node:child_process');
    const res = spawnSync('aws', ['s3', '', 'ls']);
    console.log(res.stderr.toString());"

usage: aws [options] <command> <subcommand> [<subcommand> ...] [parameters]
To see help text, you can run:

  aws help
  aws <command> help
  aws <command> <subcommand> help

aws: error: argument subcommand: Invalid choice, valid choices are:

ls                                       | website                                 
cp                                       | mv                                      
rm                                       | sync                                    
mb                                       | rb                                      
presign                                 
```

Reply:
Thanks, sloppy of me not to have a unit test for this.